src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
Calling Color.values() inside the loop causes it to create a new array each time the method is called which is not efficient. You should store the result of Color.values() in a local variable outside the loop and use that variable inside the loop.